README-ru.md

```diff
@@ -0,0 +1,182 @@
+* [Использовать Шаблоны Страниц](http://kubernetes.io/docs/contribute/style/page-templates/)
+* [Документация по Стиль Гайду](http://kubernetes.io/docs/contribute/style/style-guide/)
+* [Локализировать Kubernetes Документацию](https://kubernetes.io/docs/contribute/localization/)
+
+<!--
+## `README.md`'s Localizing Kubernetes Documentation
+ -->
+## `README.md` Локализация Kubernetes Документации
+
+<!--
+### Русский
+
+You can reach the maintainers of Korean localization at:
+
+* [Slack channel](https://kubernetes.slack.com/messages/kubernetes-docs-ko)
+ -->
+### Русский язык
+
+Вы можете найти мейнтейнеров Русской локализации по следующим ссылкам：
+
+*  ([GitHub - @dianaabv16](https://github.com/dianaabv16))
+* [Slack channel](https://kubernetes.slack.com/messages/kubernetes-docs-zh)
+
+<!--
+## Running the website locally using Docker
+ -->
+## Запустите сайт локально с помощью Docker
+
+<!--
+The recommended way to run the Kubernetes website locally is to run a specialized [Docker](https://docker.com) image that includes the [Hugo](https://gohugo.io) static website generator.
+ -->
+Рекомендуемый способ локального запуска веб-сайта Kubernetes - запуск специализированного образа [Docker] (https://docker.com), который включает генератор статических веб-сайтов [Hugo] (https://gohugo.io).
+
+<!--
+> If you are running on Windows, you'll need a few more tools which you can install with [Chocolatey](https://chocolatey.org). `choco install make`
+ -->
+> Если вы работаете c Windows, вам понадобится еще несколько инструментов, которые вы можете установить с помощью [Chocolatey] (https://chocolatey.org). `choco install make`
+
+<!--
+> If you'd prefer to run the website locally without Docker, see [Running the website locally using Hugo](#running-the-site-locally-using-hugo) below.
+ -->
+> Если вы предпочитаете запускать сайт локально без Docker, см. Раздел [Запуск сайта локально с помощью Hugo] (#running-the-site-localally using-hugo) ниже.
+
+<!--
+If you have Docker [up and running](https://www.docker.com/get-started), build the `kubernetes-hugo` Docker image locally:
+ -->
+Если у вас есть Docker [запущен и работает] (https://www.docker.com/get-started), локально создайте образ Docker `kubernetes-hugo`:
+
+```bash
+make docker-image
+```
+
+<!--
+Once the image has been built, you can run the website locally:
+ -->
+一 После того, как изображение было построено, вы можете запустить сайт локально:
+
+```bash
+make docker-serve
+```
+
+<!--
+Open up your browser to http://localhost:1313 to view the website. As you make changes to the source files, Hugo updates the website and forces a browser refresh.
+ -->
+Откройте браузер по адресу http://localhost:1313 для просмотра веб-сайта. При внесении изменений в исходные файлы Hugo обновляет веб-сайт и принудительно обновляет браузер.
+
+<!--
+## Running the website locally using Hugo
+ -->
+## 使用 Hugo 在本地运行网站
```

Comment:
This doesn't look like Russian to me! :wink:

Reply:
oops sorry thanks for that